power_consumption/config.py

```diff
@@ -20,14 +20,16 @@ class Target(BaseModel):
 
 
 class Dataset(BaseModel):
-    id: int
```

Comment:
I see you deleted id, which is indeed not required for model.
But I am a bit worried it is an int, identifiers must be strings